src/misc/support.cpp

```diff
@@ -70,6 +70,22 @@ char drive_letter(uint8_t index)
 	return 'A' + index;
 }
 
+char get_drive_letter_from_path(const char* path)
+{
+	const size_t len = strlen(path);
+	if (len < 2 || len > 3) {
```

Comment:
Why `len > 3`? Shouldn't this work for something like `C:\BLAH\STUFF.EXE` too? The name implies it should work for full paths.

Reply:
You asked me to extract this to a generic function and it probably shouldn't have been.  It's meant to fail if the user does not specify a root drive in the case of attempting to mount a CDROM drive.

Reply:
Or at least it shouldn't have been extracted without changes :smile: 

Moved the `> 3` length check to the calling code.

Reply:
> Moved the `> 3` length check to the calling code.

I think that's a nice solution to keep the helper generic 👍🏻